catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/controller/SceneController.java

```diff
@@ -135,6 +136,15 @@ public Scene copy(Scene sceneToCopy, Project dstProject) throws IOException {
 
 	public void delete(Scene sceneToDelete) throws IOException {
 		StorageOperations.deleteDir(sceneToDelete.getDirectory());
+		File projectDirectory = ProjectManager.getInstance().getCurrentProject().getDirectory();
```

Comment:
instead of adding this removal procedure to every delete here and there id rather have a cleanup once eg. on project load.